news/2024/2024-06-12-osu-catch-world-cup-2024-quarterfinals-recap.md

```diff
@@ -0,0 +1,134 @@
+
+Thanks to that win, Chile secured themselves a spot in the final 6, a result that they didn't achieve since 2020. Canada, on the other hand, will still be competing, and I'm sure are determined to make the most out of that losers' bracket run.
+
+### ::{ flag=JP }:: Japan vs. ::{ flag=GB }:: United Kingdom
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="NEED TO CHANGE LINK" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+Japan and the United Kingdom are both teams that we've rarely seen access the later rounds of the tournament in recent years. Japan managed to reach them last year for the first time since 2018 before being eliminated, finishing top 12, while the United Kingdom never made it past the Quarterfinals and the final 12 (at least since the introduction of the losers' bracket in 2015). This year though, these two teams are looking better than they have for a long time, and are looking to beat their previous results. Japan, getting seed 8, looked very promising from the very beginning, with several of their players improving a lot throughout the year. As for the United Kingdom, several of their veteran players started grinding the game again, and their newcomers keep on improving at a rapid pace. Both teams were sent to the losers' bracket in the previous week, but ended up winning their first Quarterfinals match convincingly, 6-1 against Sweden for Japan, and 6-0 against the Netherlands for the United Kingdom. Now meeting in an elimination match, one of these teams would end their CWC run here, despite their desire to keep on going and show what they're capable of.
+
+Right off the bat, we see back-to-back breakpoints, setting the tone for an intense series. For the following picks, both teams would exhaust the mod brackets they're known to be strong at: Japan would go on to pick and win [DT3](https://osu.ppy.sh/beatmapsets/2191178#fruits/4634726) and [DT1](https://osu.ppy.sh/beatmapsets/2191244#fruits/4634876), in which ::{ flag=JP }:: [Molqus](https://osu.ppy.sh/users/1927193) would get his second FC on the map, a repeat of his earlier performance against Sweden. The United Kingdom would do the same but with the Hard Rock pool, securing points on [HR2](https://osu.ppy.sh/beatmapsets/2191172#fruits/4634714) and [HR3](https://osu.ppy.sh/beatmapsets/2191179#fruits/4634727). Now that both of these mod pools were exhausted because of the first pick on [HR1](https://osu.ppy.sh/beatmapsets/2191169#fruits/4634698) by Japan and the [DT2](https://osu.ppy.sh/beatmapsets/2191261#fruits/4634911) ban from the United Kingdom, they had to try something new.
```

Comment:
```suggestion
Right off the bat, we see back-to-back breakpoints, setting the tone for an intense series. For the following picks, both teams would exhaust the mod brackets they're known to be strong at — Japan would go on to pick and win [DT3](https://osu.ppy.sh/beatmapsets/2191178#fruits/4634726) and [DT1](https://osu.ppy.sh/beatmapsets/2191244#fruits/4634876), with ::{ flag=JP }:: [Molqus](https://osu.ppy.sh/users/1927193) getting their second FC on DT3, a repeat of their earlier performance against Sweden. The United Kingdom would do the same but with the Hard Rock pool, securing points on [HR2](https://osu.ppy.sh/beatmapsets/2191172#fruits/4634714) and [HR3](https://osu.ppy.sh/beatmapsets/2191179#fruits/4634727). Now that both of these mod pools were exhausted because of the first pick on [HR1](https://osu.ppy.sh/beatmapsets/2191169#fruits/4634698) by Japan and the [DT2](https://osu.ppy.sh/beatmapsets/2191261#fruits/4634911) ban from the United Kingdom, they had to try something new.
```